justfile

```diff
@@ -65,8 +65,6 @@ _prep-steps:
   @just show > tmp/repo-status.txt
   @just analyze > tmp/analysis-report.txt
   @just build > tmp/build-output.txt
```

Comment:
Let's cut build too, I don't think we need it and it takes up a decent amount of time